utils/automation/.github-workflow.yml

```diff
@@ -11,25 +11,67 @@ jobs:
   check-license:
     name: Check license
     runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - run: |
+      - name: Checkout this repository
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Run Scanners (excluding allowlisted)
```

Comment:
```suggestion
      - name: Run Scanners (including allowlisted)
```

Reply:
`Excluding` allowlisted I thought would mean excluding the allowlisted licenses from scan. Should I change it still? It's probably a grammatical issue 🤓

Reply:
Yea yea, I think I got it in another reference. Maybe (Run Scanners) is just fine.  Like in the Copyright stage